README.md

```diff
@@ -561,6 +561,7 @@ List of all available rules. The rules ported from `golint` are left unchanged a
 | [`redundant-build-tag`](./RULES_DESCRIPTIONS.md#redundant-build-tag) | n/a   | Warns about redundant `// +build` comment lines |   no    |  no   |
 | [`use-errors-new`](./RULES_DESCRIPTIONS.md#use-errors-new) | n/a   | Spots calls to `fmt.Errorf` that can be replaced by `errors.New` |   no    |  no   |
 | [`redundant-test-main-exit`](./RULES_DESCRIPTIONS.md#redundant-test-main-exit) |  n/a   | Suggests removing `Exit` call in `TestMain` function for test files |    no    |  no   |
+| [`time-after-leak`](./RULES_DESCRIPTIONS.md#time-after-leak) |  n/a   | Suggests replacing `time.After` with `time.NewTimer` to prevent goroutine leak |    no    |  no   |
```

Comment:
I would mention here it's for older Go versions, or something like


```suggestion
| [`time-after-leak`](./RULES_DESCRIPTIONS.md#time-after-leak) |  n/a   | Suggests replacing `time.After` with `time.NewTimer` to prevent goroutine leak, if needed |    no    |  no   |
```


```suggestion
| [`time-after-leak`](./RULES_DESCRIPTIONS.md#time-after-leak) |  n/a   | Suggests replacing `time.After` with `time.NewTimer` to prevent goroutine leak for old Go versions |    no    |  no   |
```

Reply:
But maybe, one interesting thing could be to add a column to this table to mention the Go version specification

Something like < 1.23 here, and > 1.18 for use-any, and others.